src/opm/test_util/EclFilesComparator.cpp

```diff
@@ -442,8 +457,8 @@ void RegressionTest::gridCompare(const bool volumecheck) const {
                          << (active1 ? "active" : "inactive") << " in first grid, but "
                          << (active2 ? "active" : "inactive") << " in second grid.");
         }
-        const double cellVolume1 = ecl_grid_get_cell_volume1(ecl_grid1, cell);
-        const double cellVolume2 = ecl_grid_get_cell_volume1(ecl_grid2, cell);
+        const double cellVolume1 = getCellVolume(ecl_grid1, cell);
```

Comment:
There is a file name based constructor of the `EclipseGrid` class - so you could do this with:
```C++
EclipseGrid grid("FILENAME.EGRID");
```

Reply:
That would require more extensive reworking of the comparison tool I think, which was not my intended scope for this PR.